src/smk/tool/layers/layer-display.html

```diff
@@ -37,6 +37,7 @@
             <div class="smk-legend-item"
                 v-for="legend in display.legends"
             >
+                <span v-if="legend.indent">&nbsp;&nbsp;</span>
```

Comment:
Can this be accomplished with CSS?

Reply:
Yes (here in layer as well as in legend). Good point; there's no semantic meaning in a nonbreaking space, which might manifest in text-to-speech readers or other alternate tools. I've created styles and applied them.